include/swift/AST/Stmt.h

```diff
@@ -78,7 +78,6 @@ class alignas(8) Stmt {
   SourceLoc getEndLoc() const;
   
   SourceRange getSourceRange() const;
-  SourceLoc TrailingSemiLoc;
```

Comment:
I'd prefer not to remove source information from the current nodes if it's not going to get observed during parse. Also, if you are going to use `consumeToken()` call, please either comment what you are consuming, or use `consumeToken(tok::_____)` to indicate what you are expecting to drop. But I would prefer not to drop source information right now.